packages/shared/src/components/Feed.tsx

```diff
@@ -282,23 +290,27 @@ export default function Feed<T>({
     row: number,
     column: number,
   ) => {
-    onShareMenuClick(e, post, index, row, column);
-    const trackEventOptions = {
-      columns: virtualizedNumCards,
-      column,
-      row,
-      ...feedAnalyticsExtra(
-        feedName,
-        ranking,
-        undefined,
-        undefined,
-        ExperimentWinner.PostCardShareVersion,
-      ),
-    };
-    trackEvent(postAnalyticsEvent('open share', post, trackEventOptions));
-    lastShareMenuCloseTrackEvent = () => {
-      trackEvent(postAnalyticsEvent('close share', post, trackEventOptions));
-    };
+    if (copyLinkFeature) {
+      copyLink(post, index, row, column);
```

Comment:
Decided to leverage the existing prop drilling instead of having to introduce yet another fn to drill through 7 components 😅